hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -260,7 +260,8 @@ private void writeToContainerFile(File containerFile, boolean isCreate)
     } catch (IOException ex) {
       onFailure(containerData.getVolume());
       throw new StorageContainerException("Error while creating/ updating " +
-          ".container file. ContainerID: " + containerId, ex,
+          " container file. ContainerID: " + containerId +
+          " ,container path: " + containerFile.getAbsolutePath(), ex,
```

Comment:
Note that error may happen for temp file (lines 246-248) or final file (lines 254, 256-257).  Can we distinguish between those two cases?

Also, spacing is a bit odd here (some double, some missing).  How about:

```java
      throw new StorageContainerException("Error while creating/updating" +
          " container file. ContainerID: " + containerId +
          ", container path: " + containerFile.getAbsolutePath(), ex,
```

Reply:
Added additional text when the temp file could not be created. I'm not sure what else we can do with this situation.